apps/hestia/src/components/cexOnRamp/template.tsx

```diff
@@ -0,0 +1,122 @@
+"use client";
+
+import { renderSendWidget } from "@cedelabs/widgets-universal";
+import { useConnectWalletProvider } from "@orderbook/core/providers/user/connectWalletProvider";
+import { Typography } from "@polkadex/ux";
+import { RiInformation2Line } from "@remixicon/react";
+import { Fragment, useEffect, useMemo, useRef } from "react";
+import { useResizeObserver, useWindowSize } from "usehooks-ts";
+
+import { Help } from "../transfer/Help";
+import { SelectAsset } from "../transfer/SelectAsset";
+import { ConnectTradingInteraction } from "../ui/ConnectWalletInteraction/connectTradingInteraction";
+import { QuickStart } from "../ui/Footer/QuickStart";
+import { ResponsiveProfile } from "../ui/Header/Profile/responsiveProfile";
+import { themeConfig } from "../../../../../themeConfig";
+
+import { Footer, Header } from "@/components/ui";
+import { useTour, useTransfer } from "@/hooks";
+
+export function Template() {
+  const { width } = useWindowSize();
+  const { onOpenChange, open, onClose } = useTour();
+
+  const headerRef = useRef<HTMLDivElement | null>(null);
+  const helpRef = useRef<HTMLDivElement | null>(null);
+  const formwRef = useRef<HTMLDivElement | null>(null);
+  const footerRef = useRef<HTMLDivElement | null>(null);
+  const interactionRef = useRef<HTMLDivElement | null>(null);
+
+  const { height: footerHeight = 0 } = useResizeObserver({
+    ref: footerRef,
+    box: "border-box",
+  });
+
+  const { height: interactionHeight = 0 } = useResizeObserver({
+    ref: interactionRef,
+    box: "border-box",
+  });
+
+  const {
+    onChangeAsset,
+    selectedAsset,
+    assetsInteraction,
+    onAssetsInteraction,
+  } = useTransfer();
+
+  const { selectedWallet, browserAccountPresent, extensionAccountPresent } =
+    useConnectWalletProvider();
+
+  useEffect(() => {
+    renderSendWidget("#cede-widget", {
+      theme: {
+        borderRadius: -15,
+        borderColor: themeConfig.theme.extend.colors["secondary-base"],
+        inputBorderColor: themeConfig.theme.extend.colors["secondary-base"],
+        logoBorderColor: themeConfig.theme.extend.colors["primary-base"],
+        accentColor: themeConfig.theme.extend.colors["primary-base"],
```

Comment:
Please refactor it a bit with a variable. 

```js
const extend = themeConfig.theme.extend; // Define it outside the component.
```

and use it like this.

```js
borderColor: extend.colors["secondary-base"]
```

Check [this](https://github.com/Polkadex-Substrate/Polkadex-Orderbook-Frontend/blob/release/apps/hestia/src/components/trading/Graph/config.ts#L11) out for help.